internal/server/kong/ws/config/version_compatibility.go

```diff
@@ -260,6 +261,45 @@ func (vc *WSVersionCompatibility) processConfigTableUpdates(uncompressedPayload
 	return processedPayload, nil
 }
 
+func (vc *WSVersionCompatibility) removeCoreEntity(
+	processedPayload string,
+	entityType string,
+	dataPlaneVersionStr string,
+	changeID ChangeID,
+	tracker *ChangeTracker,
+) string {
+	entity := gjson.Get(processedPayload, fmt.Sprintf("config_table.%ss", entityType))
+	if entity.IsArray() {
+		for _, res := range entity.Array() {
+			var err error
+			vaultID := res.Get("id").String()
+			err = tracker.TrackForResource(changeID, ResourceInfo{
+				Type: entityType,
+				ID:   vaultID,
+			})
+			if err != nil {
+				vc.logger.Error("failed to track version compatibility change",
+					zap.String("change-id", string(changeID)),
+					zap.String("resource-type", entityType))
+			}
+		}
+	}
+
+	processedPayload, err := sjson.Delete(processedPayload, fmt.Sprintf("config_table.%ss", entityType))
+	if err != nil {
+		vc.logger.With(zap.String("entity", entityType)).
+			With(zap.String("data-plane", dataPlaneVersionStr)).
+			With(zap.Error(err)).
+			Error("entity was not removed from configuration")
+	} else {
+		vc.logger.With(zap.String("entity", entityType)).
+			With(zap.String("data-plane", dataPlaneVersionStr)).
+			Warn("removing %ss entity which is incompatible with data plane")
```

Comment:
```suggestion
			Warn("removing entity which is incompatible with data plane")
```

No need to add the entity type here, as that's already part of the logger `zap.String("entity", entityType)`

Reply:
I made these changes in #483